metals/src/main/scala/scala/meta/internal/metals/debug/DebugProxy.scala

```diff
@@ -110,6 +114,20 @@ private[debug] final class DebugProxy(
       }
       response.setResult(args.toJson)
       client.consume(response)
+    case message @ ErrorOutputNotification(output) =>
+      Try {
+        stackTraceAnalyzer
+          .fileLocationFromLine(output.getOutput())
+      }.toOption.flatten
```

Comment:
I may be totally missing something, but `fileLocationFromLine` here just returns an `Option` and `getOutput` a `String`. Why do we need to wrap in a `Try` here?

Reply:
It could actually throw still from substring. Fixed it to not throw and left it with option. Good point!

Reply:
Ahh true, missed the substring. 👍